samples/visual_indicator/nodejs/README.md

```diff
@@ -0,0 +1,129 @@
+---
+page_type: sample
```

Comment:
Kindly give proper naming:
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/59052547/15a05d59-1de0-4c73-86b3-154a095a669c)


1) 2.Visual-Indicator
2) Visual.Indicator.gif   etc..

Update in Readme and sample.json